src/Form/FeatureNotificationType.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\FeatureNotification;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\Extension\Core\Type;
+use Symfony\Component\Validator\Constraints;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\CallbackTransformer;
+
+class FeatureNotificationType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('status', Type\CheckboxType::class, [
+                'label' => false,
+                'required' => false,
+                'attr' => [
+                    'data-toggle' => 'toggle',
+                    'data-on' => 'Enable',
+                    'data-off' => 'Disable',
+                    'data-onstyle' => 'success'
+                ]
+            ])
+            ->add('title', Type\TextType::class, [
+                'label' => 'Title',
+                'required' => true,
+                'constraints' => [
+                    new Constraints\NotBlank(),
+                    new Constraints\Type('string')
+                ]
+            ])
+            ->add('message', Type\TextareaType::class, [
+                'required' => true,
+                'attr' => ['rows' => 4],
+                'constraints' => [
+                    new Constraints\NotBlank(),
+                    new Constraints\Type('string')
+                ]
+            ])
+            ->add('new_feature_status', Type\CheckboxType::class, [
```

Comment:
Do we need a separate field for this? Could it just be derived from whether or not there is a URL?

Reply:
I had the same doubt when working on this but I just implemented as it's mentioned in the mockup. I'll check with Gwen and see what she says.